src/vaadin-select-text-field.html

```diff
@@ -27,16 +28,38 @@
         return 'vaadin-select-text-field';
       }
 
+      static get properties() {
+        return {
+          _valueId: String
+        };
+      }
+
       static get template() {
-        // Check if text-field is using slotted input
-        if (super.template.content.querySelector('slot[name="input"]')) {
-          return super.template;
```

Comment:
The idea of using `slot[name="input"]` was to avoid having an extra slot and reuse existing one. So maybe this code should not be removed, but updated.

Also, it was implemented this way to ensure that `vaadin-select` works with older `vaadin-text-field` versions (which do not support the slotted input).